test/test_translator.py

```diff
@@ -155,6 +230,23 @@ def test_get_road_direction():
 
     assert test_direction==valid_direction
 
+def test_get_road_direction_empty_coordinates():
```

Comment:
We have several distinct tests within this method. The first is testing invalid data, 2nd an empty coordinate set, and 3rd a null coordinate set. These should each be separate tests to show the functionality.

Reply:
i think i separeted it and named it 
1.test_get_road_direction_null_coordinates()
2.test_get_road_direction_empty_coordinates()
3.test_get_road_direction_empty_string():